### DISCUSSION TITLE
Need help with homebrew pip issue 5599 on macOS Big Sur

### DISCUSSION THREAD

**Discussion by bbvs247** [Homebrew/discussions#1432] [Getting started]
Hey, 

(I'm new here, I used to use Reddit to ask questions, so please don't blame me if this is the wrong category or something. I'm doing the best I can)
When using topgrade in homebrew on macOS big sur on a 2020 Intel 13" default upper specs MacBook Pro I ran into the following issue: `xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun`

Looking for a solution on the web I found: https://andreaolivato.medium.com/how-to-solve-the-invalid-active-developer-path-brew-error-after-updating-to-mac-os-catalina-10-15-e351e743dd23

Following the advice I ran: `xcode-select --install` and restarted afterwards.

After running topgrade again I became a bit suspicious when it said: 
```console
Successfully installed pip-21.1.1
WARNING: You are using pip version 19.2.3, however version 21.1.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```

but I though well, seems like it updated itself. So I ran tograde again and it said 
```console
"WARNING: pip is being invoked by an old script wrapper. This will fail in a future version of pip.
Please see https://github.com/pypa/pip/issues/5599 for advice on fixing the underlying issue.
To avoid this problem you can invoke Python with '-m pip' instead of running pip directly.
Requirement already satisfied: pip in ./Library/Python/3.8/lib/python/site-packages (21.1.1)"
```
I followed the link but found no solution other than the python workaround, wich seems to be a short term solution as well. As far as I now have both pip version 19.2.3 as well as 21.1.1 and need to remove the old one right? But how do I do that?
And does "Requirement already satisfied: pip in ./Library/Python/3.8/lib/python/site-packages (21.1.1)" mean it already fixed itself? And what happens if I ignore the problem? If anyone could help me out it would really be appreciated. While I love tech I've got no coding experience at all in this is just over my head. That you so much!

**Comment by SMillerDev** [comment]
How did you install topgrade? Because in all of this I don't see any reference to `brew` or the Homebrew install path.

**Reply by bbvs247** [reply]
I used "brew install topgrade" after installing homebrew on my device. I though all of this was therefore based on homebrew, especially as I use topgrade to keep my via brew install installed programs (and the rest) up to date.
